projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -122,4 +263,53 @@ public void stateObjective(String projectId, String objective) {
   }
 
 
+  /**
+   * Retrieve all analytes of the active experiment. If no experiment is active, returns an empty
+   * list.
+   *
+   * @param projectId the project the experiment belongs to
+   * @return a collection of analytes in the active experiment. If no experiment is active, returns
+   * an empty list.
+   */
+  public Collection<Analyte> getAnalytesOfActiveExperiment(ProjectId projectId) {
+    return projectRepository.find(projectId)
+        .map(project ->
+            loadActiveExperimentForProject(project).map(Experiment::getAnalytes)
+                .orElse(List.of()))
+        .orElseThrow(() -> new IllegalArgumentException("Project could not be retrieved."));
+  }
```

Comment:
For what case do we need this service interface?

Reply:
If we would not have this interface we would need to provide the UI with the experiment entity. This entity is part of the project aggregate and must not be exposed.